src/MagnumPlugins/BasisImporter/Test/README.md

```diff
@@ -0,0 +1,16 @@
+Creating input Basis files
+==========================
+
+The basis test files were converted from the PngImporter's test files
+`rgb.png`, `rgba.png` (and 90° clockwise rotated version for the second images
+for the `rgb(a)_2_images.basis` files) using the official basis universal
+[conversion tool](https://github.com/BinomialLLC/basis_universal/#command-line-compression-tool).
```

Comment:
Could you craft larger files (64x32 at least, for example)? Those two are 3x2 and probably encode just into random noise, as GPU compression blocks are at least 4x4 pixels. Not sure if block-aligning is a good idea, so maybe 63x27 or some such :)

There's this now: https://github.com/mosra/magnum/tree/master/src/Magnum/Shaders/Test/TestFiles feel free to use/resize/crop any of those as you see fit (some are quite dark tho, so check first). The `magnum-imageconverter` can convert them to PNGs, or create a new one from the SVG.

Reply:
Replace all of the existing files with larger files?